### ISSUE TITLE
use_quantiles description in canny is incorrect

### ISSUE THREAD

**Issue by pigunther** [scikit-image/scikit-image#4282]
## Description
The description of `use_quantiles` is "If True then treat low_threshold and high_threshold as quantiles of the edge magnitude image, rather than absolute edge magnitude values. **If True then the thresholds must be in the range [0, 1].**"

However, he thresholds must be in the range [0, maximum of the image's dtype ]

## Way to reproduce
```python
from skimage import data
from skimage.feature import canny
image = data.camera()
edge_canny = canny(image, low_threshold=50, high_threshold=150, use_quantiles=True)
# No error is produced
```

## Cause
This happens because the thresholds are divided by maximum of the image's dtype (lines 163, 168 in [_canny.py](https://github.com/scikit-image/scikit-image/blob/master/skimage/feature/_canny.py))

## Probable solution
The probable solution is not to divide the thresholds if `use_quantiles` is True or just simpler change the description.

## Version information
```python
# your output here
3.7.3 (default, Sep  5 2019, 17:14:41) 
[Clang 11.0.0 (clang-1100.0.33.8)]
Darwin-19.0.0-x86_64-i386-64bit
scikit-image version: 0.16.2
numpy version: 1.17.3
```

**Comment by jni** [comment]
Yikes! Thanks for the report, @pigunther! I think you are right that we should not divide by the dtype max.

**Comment by rfezzani** [comment]
To fix this issue, I propose in #4342 to add the `preserve_range` argument and to divide the thresholds only if `preserve_range` is True.

**Comment by rfezzani** [comment]
A simpler bug fix is proposed in #4437.
